### PR TITLE
Fix local artifact recognition and usage by SHARK downloader.

### PR DESCRIPTION
**Pull request by monorimet**

### CODE REVIEW

File: generate_sharktank.py [hunk 1]
```diff
@@ -28,7 +29,8 @@
     pass
 
 # All generated models and metadata will be saved under this directory.
-WORKDIR = os.path.join(os.path.dirname(__file__), "gen_shark_tank")
+home = str(Path.home())
+WORKDIR = os.path.join(home, ".local/shark_tank/")
```
Comment (AmosLewis) [inline generate_sharktank.py:33]: line 242        
os.system(
            "gsutil cp -r ./gen_shark_tank/* gs://shark_tank/" + git_hash
        )

**Review by AmosLewis — CHANGES_REQUESTED** [review]

File: generate_sharktank.py (hunk 1 above)
Comment (monorimet) [reply generate_sharktank.py:33]: thanks!

File: shark/shark_downloader.py [hunk 2]
```diff
@@ -110,7 +110,9 @@ def gs_download_model():
             np.load(os.path.join(model_dir, "upstream_hash.npy"))
         )
         if local_hash != upstream_hash:
-            gs_download_model()
```
Comment (dan-garvey) [inline shark/shark_downloader.py:113]: how does model downloading occur if not when the hash is nonmatching?

Comment (monorimet) [reply shark/shark_downloader.py:113]: If ~/.local/shark_tank/ is deleted, as directed by the print statement in check_dir_exists

**Review by AmosLewis — COMMENTED** [review]
LGTM

**Review by AmosLewis — APPROVED** [review]

File: shark/shark_downloader.py (hunk 2 above)
Comment (dan-garvey) [reply shark/shark_downloader.py:113]: can we add a flag that disables the hash auto download and leave it enabled by default?

**Pull request merged**